src/Infrastructure/Form/Map/MapFilterFormType.php

```diff
@@ -14,6 +14,24 @@ final class MapFilterFormType extends AbstractType
     public function buildForm(FormBuilderInterface $builder, array $options): void
     {
         $builder
+        ->add(
+            'displayMeasureTypeNoEntry',
+            CheckboxType::class,
+            options: [
+                'label' => 'map.filter.type.noEntry',
+                'value' => 'yes',
+                'required' => false,
+            ],
+        )
+        ->add(
+            'displayMeasureTypeSpeedLimitation',
+            CheckboxType::class,
+            options: [
+                'label' => 'map.filter.type.speedLimitation',
+                'value' => 'yes',
+                'required' => false,
+            ],
+        )
```

Comment:
Ici on ferait donc plutôt qqc du genre :

```suggestion
        ->add(
            'measureTypes',
            ChoiceType::class,
            options: [
                'label' => 'map.filter.measureType
                'choices' => MeasureTypeEnum::values(),
                'required' => false,
            ],
        )
```

(c'est probablement pas bon en l'état, il faut reprendre l'exemple des `critairTypes`)